uiowa/uiowa
# Feature: SPPA opportunity resource migrations

Resolves https://github.com/uiowa/uiowa/issues/3722. 

## To Test

- Sync D7 sppa (urban) site within grad application using Dev Desktop.
- Run the following and configure/save SiteNow Migrate
```
blt sync --site sppa.uiowa.edu && 
drush @sppa.local en sppa_migrate --yes && 
drush @sppa.local uli /admin/config/system/sitenow-migrate
```
```
database = urban
user = root
host = 10.0.2.2
port = 33067
base url = https://sppa.uiowa.edu
```
- `drush @sppa.local cr && drush @sppa.local migrate:status`
- You should see potential employers and consulting firms migrations
`drush @sppa.local migrate:import --all`
- Test that redirects are working: https://sppa.local.drupal.uiowa.edu//content/ad-marble-and-company, https://sppa.local.drupal.uiowa.edu/potential-employer/university-of-virginia-batten-school-of-leadership-and-public-policy
